File: mytestvueapp.client/src/entities/Notification.ts
export default class notification {
  artId: number;
  artistId: number;
  commentId: number;
  type: number; //0 for comment, 1 for likes
  user: string; //user who did the action
  viewed: boolean;
  artName: string;

  constructor() {
    this.artId = -1;
    this.artistId = -1;
    this.commentId = -1;
    this.type = -1;
    this.user = "";
    this.viewed = false;
    this.artName = "";
  }
}

File: MyTestVueApp.Server/Entities/Notification.cs
namespace MyTestVueApp.Server.Entities
{
    public class Notification
    {
        public int CommentId { get; set; }
        public int ArtistId { get; set; }
        public int ArtId { get; set; }
        public int Type { get; set; }
        public string User { get; set; }
        public bool Viewed { get; set; }
        public string ArtName { get; set; }
    }
}
